pkg/test/cmd/extension/command.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2021 Tetrate
```

Comment:
If this is only for tests, I suggest we suffix `_test.go` so it can not be compiled in the last binary

Reply:
sgtm

Reply:
I realized that this package is contained in the `pkg/test` dir. so I think this is fine as it is!